cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
In the `validate` method, ensure that the keys 'row', 'seat', and 'movie_session' exist in `attrs` before accessing them to avoid potential `KeyError`. Consider using `attrs.get('key')` with a default value or adding a check for key existence.